README.md

```diff
@@ -6,6 +6,10 @@ _Write once, run on any cloud ☁️_
 [![godoc](https://godoc.org/github.com/google/go-cloud?status.svg)][godoc]
 [![Coverage](https://codecov.io/gh/google/go-cloud/branch/master/graph/badge.svg)](https://codecov.io/gh/google/go-cloud)
 
+<p align="center">
+  <img width="509" height="276" src="internal/website/static/go-cdk-logo-gopherblue.png">
```

Comment:
Make sure to include `alt=""` to mark the image as purely decorative and not necessary for screenreaders to read.

```suggestion
  <img width="509" height="276" src="internal/website/static/go-cdk-logo-gopherblue.png" alt="">
```